bindings/templates/src/main/resources/typescript-fetch-globals/modelEnum.mustache

```diff
@@ -0,0 +1,12 @@
+/**
+ * {{{description}}}
+ * @export
+ * @enum {string}
+ */
+export enum {{classname}} {
+{{#allowableValues}}
+{{#enumVars}}
+    {{{name}}} = <any> {{{value}}}{{^-last}},{{/-last}}
+{{/enumVars}}
+{{/allowableValues}}
+}
```

Comment:
FYI: we had a lengthy convo in late 2022 about moving enums to a const -- I would say it's best we get this PR merged and then I'll probably recc. bringing in [this mustache file](https://github.com/determined-ai/determined/pull/5533/files#diff-084ad1859484550b5e7fb2f8ce1e856ba37f627c5618729356155871625d31db)  - (this causes changes across web UI, so better not to roll it in now)